db/migrations/0003_cinemahall_rows_cinemahall_seats_in_row_and_more.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.2 on 2023-08-13 22:47
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='cinemahall',
+            name='rows',
+            field=models.IntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name='cinemahall',
+            name='seats_in_row',
+            field=models.IntegerField(default=0),
+        ),
+        migrations.AddField(
+            model_name='movie',
+            name='description',
+            field=models.TextField(null=True),
+        ),
+    ]
```

Comment:
It is considered a better practice to include one migration per module in a PR, generally, there is no need to commit multiple of them. Include only one migration here.